openstack/designate/templates/central-deployment.yaml

```diff
@@ -44,7 +44,7 @@ spec:
             - name: DEPENDENCY_JOBS
               value: "designate-migration"
             - name: DEPENDENCY_SERVICE
-              value: "designate-mariadb,{{ .Release.Name }}-rabbitmq"
```

Comment:
same as above - plus that central does not have the middleware

Reply:
removed the change.